### PR TITLE
[@types/ioredis] hgetall returns object

### PR DESCRIPTION
**Pull request by massivefermion**
changed type of hgetall result from string[] to object

### CODE REVIEW

File: types/ioredis/index.d.ts [hunk 1]
```diff
@@ -323,8 +323,8 @@ declare namespace IORedis {
         hvals(key: KeyType, callback: (err: Error, res: string[]) => void): void;
         hvals(key: KeyType): Promise<string[]>;
```
Comment (SimonSchick) [inline types/ioredis/index.d.ts]: That should be `{ [key: string]: string | number }` iirc

Comment (MassiveFermion) [reply types/ioredis/index.d.ts]: fixed it

**Review by SimonSchick — APPROVED** [review]

**Comment by orta** [comment]
Cool, looks good to me

**Comment by demian85** [comment]
Can someone please explain this commit: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40966/commits/f3e255236d34ab03de32dd3581c0737575955311

Why does `hgetall` return `string | number`? Redis stores all its data as strings, it just makes no sense for a key to have a number type. It is breaking my types. Did anyone validate this before approving such thing?

File: types/ioredis/index.d.ts (hunk 1 above)
Comment (demian85) [reply types/ioredis/index.d.ts]: This is completely wrong. Should I create another pull request fixing an already merged pull request?

Comment (SimonSchick) [reply types/ioredis/index.d.ts]: No you would open a new one.
My bad for suggesting the wrong type here, got it mixed up with something else.

**Pull request merged**